activesupport/lib/active_support/testing/file_fixtures.rb

```diff
@@ -0,0 +1,35 @@
+module ActiveSupport
+  module Testing
+    # Adds simple access to sample files called file fixtures.
+    # File fixtures are normal files stored in
+    # <tt>ActiveSupport::TestCase.file_fixture_directory</tt>.
+    module FileFixtures
+      extend ActiveSupport::Concern
+
+      included do
+        class_attribute :file_fixture_directory
```

Comment:
I intentionally chose a different name because there is a `file_fixture_path` method. Thought it would only make it harder to hold them apart...